### PR TITLE
importante aporte sobre el v-for y el v-if

### PR DESCRIPTION
**Pull request by carlos00027**
en la documentación oficial se encuentra esta importante recomendación

### CODE REVIEW

File: src/v2/guide/list.md [hunk 1]
```diff
@@ -440,6 +440,10 @@ De forma similar a la plantilla `v-if`, también puede usar una etiqueta `<templ
 
 ## `v-for` con `v-if`
 
+nota: Tenga en cuenta que no se recomienda usar v-if y v-for juntos. Consulte la [guía de estilo](https://vuejs.org/v2/style-guide/#Avoid-v-if-with-v-for-essential) para más detalles.
+
```
Comment (sxcamacho) [inline src/v2/guide/list.md:444]: Remember to keep a single blank line between the texts, adding the class `tip` should be displayed correctly.

File: src/v2/guide/list.md [hunk 2]
```diff
@@ -440,6 +440,10 @@ De forma similar a la plantilla `v-if`, también puede usar una etiqueta `<templ
 
 ## `v-for` con `v-if`
 
+nota: Tenga en cuenta que no se recomienda usar v-if y v-for juntos. Consulte la [guía de estilo](https://vuejs.org/v2/style-guide/#Avoid-v-if-with-v-for-essential) para más detalles.
```
Comment (sxcamacho) [inline src/v2/guide/list.md:443]: Hey @carlos00027, thanks for noticing this new content! Please, can you update this PR to include the HTML and markup?

```suggestion
<p class="tip">Tenga en cuenta que **no se recomienda** usar `v-if` y `v-for` juntos. Consulte la [guía de estilo](/v2/style-guide/#Avoid-v-if-with-v-for-essential) para más detalles.</p>
```

**Review by sxcamacho — CHANGES_REQUESTED** [review]

**Comment by Tintef** [comment]
@sxcamacho Maybe one of us should make this changes, what do you think? (It's been like 2 months since this PR was opened)

**Comment by sxcamacho** [comment]
Hey @Tintef, yes... would be great If @carlos00027 could complete it, but If he can't we can tackle it.

**Review by sxcamacho — APPROVED** [review]

**Pull request merged**